client/src/views/CodeView.vue

```diff
@@ -114,6 +117,7 @@ export default {
       const duckDbRsp = await axios.post("/api/parse/", {
         query: this.code,
       });
+      store.set_plan(duckDbRsp.data);
```

Comment:
Is there any harm in allowing download from JSON if the user uploaded from JSON?